reproducibility_project/src/engines/mcccs/project.py

```diff
@@ -176,15 +186,19 @@ def equil_replicate_set(job):
 @Project.label
 def replicate_set(job):
     """Check if number of replicates for prod has been set."""
-    return job.doc.get("num_prod_replicates") == 4
+    return (job.doc.get("num_prod_replicates") == 4) and isinstance(
+        job.doc.get("prod_replicates_done"), int
+    )
 
 
 @Project.label
 def all_prod_replicates_done(job):
     """Check if all prod replicate simulations completed."""
     try:
-        a = job.doc.prod_replicates_done
-        b = job.doc.num_prod_replicates
+        a = job.doc.get("prod_replicates_done")
+        b = job.doc.get("num_prod_replicates")
+        if a >= b:
+            print("simulation complete for {} job".format(job))
```

Comment:
Im not sure if `NoneType` will work in a comparison like this. the `get` method will return `None` by default if one of those values are missing. If 0 is the lowest the number can be, I recommend:
```suggestion
        a = job.doc.get("prod_replicates_done", 0)
        b = job.doc.get("num_prod_replicates", 0)
        if a >= b:
            print("simulation complete for {} job".format(job))
```

Reply:
So in case the `prod_replicates_done` is not stored in job doc, will `job.doc.get("prod_replicates_done", 4)` return 4?

Reply:
Correct! The second value is the value that is returned if `get` cannot find that entry.

Reply:
Thanks